source/pmc/responsibilities.md

```diff
@@ -0,0 +1,88 @@
+---
+title: PMC Responsibilities
+---
+
+The Project Management Committee (PMC) is responsible for the oversight
+of the project, including technical decisions, ensuring that the project
+is operating in accordance with ASF norms, adding new members to the
+project, and voting on releases.
+
+* [Conducting business](#conducting-business)
+* [Ensuring project health](#ensuring-project-health)
+* [Adding community members](#adding-community-members)
+* [Reporting](/pmc/reporting)
+
+## Conducting Business
+
+The PMC is expected to conduct all of its business on the public
+developers mailing list, in the full view of the community.
+
+The private@ list is for communication about private matters, such as
+discussion of proposed committers, proposed new PMC members, and
+communication from the Board of Directors. Do not use it for discussion
+of features, roadmap, or other technical decisions.
+
+The Board of Directors will occasionally send you comments or queries,
+often related to your quarterly board report. Any PMC member can, and
+should, respond to these. You should not wait for the Chair to respond
+to these queries.
+
+The PMC is tasked with making decisions on the direction of the project,
+They can, and should, also incorporate the voice of the larger community
+in these decisions. However, if a vote is taken, only the votes of the
+PMC are binding on the outcome. Other community members may vote as
+well, but those votes are advisory only, and not binding.
+
+A PMC member may veto a technical decision, such as a proposed patch.
+However, they must be able to back this veto up with a technical reason,
+and be willing to discuss possible resolutions to their objection.
+
+Further discussion of voting may be found on the [Foundation
+website](https://www.apache.org/foundation/voting.html).
+
+## Making a release
+
+The process for releasing a official Apache software artifact may be
+found on the [Infra
+website](https://infra.apache.org/release-publishing.html). As a PMC
+member, you are expected to read and understand that process.
+
+When you vote on a release, it should be an indication that you have
```

Comment:
I love that part. We've been informally doing it all the times but sometimes it was just `+1` - having it spelled here is  a good best practice to have.